Dockerfile

```diff
@@ -1,5 +1,4 @@
 FROM lifflander1/vt:alpine-final
-MAINTAINER Jonathan Lifflander <jliffla@sandia.gov>
```

Comment:
possible replacement:
`LABEL maintainer="jliffla@sandia.gov"`

https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

Reply:
I'm not too worried about annotations on this container.